common/install.sh

```diff
@@ -108,21 +109,18 @@ if $FUNCTION; then
   else
     ui_print " "
     ui_print "   Enabling overlay features..."
-    sed -i -e 's/ro.boot.vendor.overlay.theme/# ro.boot.vendor.overlay.theme/g' $INSTALLER/common/system.prop
+    sed -i 's/ro.boot.vendor.overlay.theme/# ro.boot.vendor.overlay.theme/g' $INSTALLER/common/system.prop
     rm -rf $INSTALLER/system/vendor/overlay/Pixel
     rm -rf /data/resource-cache
-    rm -rf /data/dalvik-cache
-    ui_print "   Dalvik-Cache has been cleared!"
-    ui_print "   Next boot may take a little longer to boot!"
   fi
 else
   ui_print " "
   ui_print "   Disabling Pixel accent and overlay features..."
-  sed -i -e 's/ro.boot.vendor.overlay.theme/# ro.boot.vendor.overlay.theme/g' $INSTALLER/common/system.prop
-  rm -rf $INSTALLER/system/vendor/overlay/Pixel
+  sed -i 's/ro.boot.vendor.overlay.theme/# ro.boot.vendor.overlay.theme/g' $INSTALLER/common/system.prop
   rm -f $INSTALLER/system/vendor/overlay/Pix3lify.apk
-  rm -rf /data/resource-cache
+  rm -rf $INSTALLER/system/vendor/overlay/Pixel
   rm -rf /data/dalvik-cache
+  rm -rf /data/resource-cache
   ui_print "   Dalvik-Cache has been cleared!"
   ui_print "   Next boot may take a little longer to boot!"
```

Comment:
Can we say something like ``Next boot may take a little longer``, but not because that sounds.. off?